packages/fx-core/tests/plugins/solution/debug/solution.debug.scaffolding.test.ts

```diff
@@ -161,6 +161,43 @@ describe("solution.debug.scaffolding", () => {
 
         await assertLocalDebugLocalEnvs(v2Context, inputs, parameter.numLocalEnvs);
       });
+
+      it(`happy path: tab without function (${parameter.programmingLanguage}) and AAD`, async () => {
+        const projectSetting = {
+          appName: "",
+          projectId: uuid.v4(),
+          solutionSettings: {
+            name: "",
+            version: "",
+            hostType: "Azure",
+            capabilities: ["Tab"],
+            activeResourcePlugins: ["fx-resource-simple-auth"],
```

Comment:
Is it possible that simple auth is enabled but aad not?

Reply:
We don't expect this scenario since simple auth has dependency on AAD.

Reply:
So... better to make this case more real - removing `"fx-resource-simple-auth"` here

Reply:
Fixed. Thanks!